Representable/representable
# display population data on map drawing

closes #357 
**changes**
- users can now see the population of the map when drawing their community (queries census api)
- displays "Community population: ..." while loading

**to test**
- `vim ~/.bash_profile` -- insert `export CENSUS_API_KEY:"{key}"` with the census api key (reach out to me if you don't have the key - in bitwarden as well)
- `source ~/.bash_profile` to reload key
- run server, go to entry page
- check population appears when selecting an area (takes longer depending on size of community -- will work on efficiency at a later time)
- check no js errors

**screenshots**
![image](https://user-images.githubusercontent.com/41943646/91119476-9bd46800-e6d6-11ea-9845-5a1e24bc539a.png)

**other considerations**
- currently, reloading the entry page after drawing a community means the community is erased, but the population number still shows. this is fixed in #383 , so shouldn't be an issue after merging
